libs/processor_core.py

```diff
@@ -78,6 +78,7 @@ def _serve(self):
                     self._engine.stop()
                     del self._tasks[Commands.PROCESS_VIDEO_CFG]
                     logger.info("processing stopped")
+                    self.config.reload()
```

Comment:
Why is this called in STOP? Wouldn't it be better to do this right before we create an EngineConfig in `if cmd_code == Commands.PROCESS_VIDEO_CFG`?